modules/contact/src/ReferenceResidualProblem.C

```diff
@@ -18,6 +18,10 @@ InputParameters
 validParams<ReferenceResidualProblem>()
 {
   InputParameters params = validParams<FEProblem>();
+  params.addClassDescription("Convergence criteria helper which defines convergence with respect "
```

Comment:
Problem that checks for convergence relative to a user-supplied reference quantity rather than the initial residual